include/grpc/census.h

```diff
@@ -424,8 +495,8 @@ extern census_aggregation_ops census_agg_window;
     construction via census_define_view(). */
 typedef struct {
   const census_aggregation_ops *ops;
-  const void *
```

Comment:
clang_format.... (I think I fixed a typo or added the comment here, and it pushed it over 80 char line.